front/src/styles/scss/applications/stdcm/_home.scss

```
@@ -51,9 +51,28 @@
       .stdcm-consist-container {
         width: 322px;
 
+        .stdcm-consist-title {
+          padding-right: 12px;
+          position: relative;
+        }
+
+        .stdcm-consist-title::after {
+          content: '';
+          background: linear-gradient(270deg, rgba(247, 246, 238, 0) 0%, rgb(235, 235, 234) 40%);
```

I think it's possible to simplify the first color code

```suggestion
    background: linear-gradient(270deg, transparent 0%, rgb(235, 235, 234) 40%);
```

---

nice catch, fixed